conf/application.conf

```diff
@@ -22,7 +22,7 @@ accessibility-statement.service-path = "/estates"
 
 play.http.router=prod.Routes
 
-play.filters.headers.contentSecurityPolicy = "base-uri 'self';block-all-mixed-content;child-src 'self';connect-src 'self' localhost:9032 localhost:9250 localhost:12345 https://www.google-analytics.com https://www.googletagmanager.com;default-src 'self' 'unsafe-inline' 'unsafe-eval' localhost:9032 localhost:9310;font-src data: https://fonts.gstatic.com;form-action 'self' localhost:*; frame-ancestors 'self';img-src 'self' localhost:9032 data: https://www.google-analytics.com https://www.googletagmanager.com http://www.googletagmanager.com https://tagmanager.google.com https://ssl.gstatic.com https://www.gstatic.com;script-src 'self' 'unsafe-inline' 'unsafe-eval' localhost:9032 localhost:9250 localhost:12345 https://www.google-analytics.com https://www.googletagmanager.com http://www.googletagmanager.com https://tagmanager.google.com https://ssl.gstatic.com https://www.gstatic.com;style-src 'self' 'unsafe-inline' localhost:9032 https://fonts.googleapis.com;object-src 'none'; report-uri http://localhost:8822/register-an-estate/csp/report"
+play.filters.headers.contentSecurityPolicy = "base-uri 'self';block-all-mixed-content;child-src 'self';connect-src 'self' localhost:9032 localhost:9250 localhost:12345 https://www.google-analytics.com https://www.googletagmanager.com;default-src 'self' 'unsafe-inline' 'unsafe-eval' localhost:9032 localhost:9310;font-src data: https://fonts.gstatic.com;form-action 'self' localhost:*; frame-ancestors 'self';img-src 'self' localhost:9032 data: https://www.google-analytics.com https://www.googletagmanager.com http://www.googletagmanager.com https://tagmanager.google.com https://ssl.gstatic.com https://www.gstatic.com;script-src 'self' 'unsafe-inline' 'unsafe-eval' localhost:9032 localhost:9250 localhost:12345 https://www.google-analytics.com https://www.googletagmanager.com http://www.googletagmanager.com https://tagmanager.google.com https://ssl.gstatic.com https://www.gstatic.com;style-src 'self' 'unsafe-inline' localhost:9032 https://fonts.googleapis.com;object-src 'none'; report-uri http://localhost:8822/register-an-estate/csp/report code.jquery.com object-src 'none';"
```

Comment:
@CarolineS-QA  This won't be right now as the added code.jquery.com and object-src are in the wrong position of the directive.
We will have to allow code.jquery.com in the script-src and possible other directives